toc/rfc/rfc-draft-role-change-process.md

```diff
@@ -0,0 +1,94 @@
+# Meta
+[meta]: #meta
+- Name: Role Change Process
+- Start Date: 2022-04-13
+- Author(s): @gerg
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (fill in with PR link after you submit it)
+
+
+## Summary
+
+Define the process for granting or revoking roles.
+
+## Problem
+
+There are currently
+[descriptions](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md)
+for the criteria necessary to become a Contributor or Approver. It is not
+currently defined how people apply for or are granted these roles. This makes it
+difficult for community members to know how to apply for a role promotion and
+difficult for Working Group leadership to manage Contributor and Approver
+candidates.
+
+In addition, there is no defined process for removing people from Contributor or
+Approver roles if they are no longer interested in participating at that level.
+
+## Proposal
+
+### Promotion to Contributor
+
+- When a person meets the criteria to be a Contributor as defined in
+[ROLES.md](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md),
+they may submit a PR adding themselves to the organization in
+[cloudfoundry.yml](https://github.com/cloudfoundry/community/blob/main/org/cloudfoundry.yml).
```

Comment:
the working group charter should be the source of truth when it comes to roles (Approver and above).

Reply:
Addressed in https://github.com/cloudfoundry/community/pull/248/commits/16783a0d52e15c523b7f05e0d8eab200e08705a1